client/app/components/flowchart/flowchart_directive.js

```diff
@@ -82,7 +82,7 @@ angular.module('flowChart', ['dragging'] )
 // it is painful to unit test a directive without instantiating the DOM
 // (which is possible, just not ideal).
 //
-.controller('FlowChartController', ['$scope', 'dragging', '$element', function FlowChartController($scope, dragging, $element) {
+.controller('FlowChartController', ['$scope', 'dragging', '$element', '$document', function FlowChartController($scope, dragging, $element, $document) {
```

Comment:
oh yeah totally! was saying the use of it in the directive appeared to also be easily swap-able with vm